src/java.security.jgss/windows/classes/sun/security/krb5/internal/tools/Ktab.java

```diff
@@ -434,11 +454,13 @@ void printHelp() {
         System.out.println();
         System.out.println("-l [-e] [-t]\n"
                 + "    list the keytab name and entries. -e with etype, -t with timestamp.");
-        System.out.println("-a <principal name> [<password>] [-n <kvno>] [-append]\n"
+        System.out.println("-a <principal name> [<password>] [-s <salt>] [-n <kvno>] [-f | -s <salt>] [-append]\n"
```

Comment:
The first "[-s <salt>]" looks redundant? Hmm, I edited your CSR from [-s <salt>] to [-s <salt> | -f]. Maybe you want to change that to be consistent with what you have here.

Reply:
Oops, no idea why I typed twice. Thanks for fixing the CSR.